tensorboard/plugins/audio/summary_test.py

```diff
@@ -250,6 +250,39 @@ def test_default_step(self):
             # Reset to default state for other tests.
             tf2.summary.experimental.set_step(None)
 
+    def test_lengths(self):
+        data = np.array([[[1.0], [0.0]], [[1.0], [-1.0]]]).astype(np.float32)
+        lengths = np.array([1, 2]).astype(np.int32)
+        event = self.audio_event("a", data, 44100, step=333, lengths=lengths)
+        self.assertEqual(event.step, 333)
+        self.assertLen(event.summary.value, 1)
+        tensor = tf.io.parse_tensor(
+            event.summary.value[0].tensor.SerializeToString(),
```

Comment:
Can we use `tensor_util.make_ndarray(pb.value[0].tensor)` instead? (See
other tests in this file for examples.)

Reply:
Done.